packages/atomic/.storybook/default-story-shared.tsx

```diff
@@ -11,15 +11,32 @@ export const ADDON_PARAMETER_KEY = 'shadowParts';
 export interface DefaultStoryAdvancedConfig {
   engineConfig?: Partial<SearchEngineConfiguration>;
   additionalMarkup?: () => TemplateResult;
+  additionalChildMarkup?: () => TemplateResult;
+  parentElement?: () => HTMLElement;
```

Comment:
It feels a bit weird to me to have three parameters so we can have children, parents and siblings. And it feels weirder to me that one of those takes an element directly, but not the others. One caveat with the `parentElement` element right now, for instance, is that we can't pass a deeper ancestry tree than 1.

Is there a way that we could support a config like the following, where one element would be replaced with the configurable element?
```jsx
// atomic-timeframe.stories.tsx
const {defaultModuleExport, exportedStory} = defaultStory(
  'Atomic/TimeframeFacet/Timeframe',
  'atomic-timeframe',
  {unit: 'year'},
  TimeframeDoc,
  {
    markup: html```
      <style>
        atomic-timeframe-facet {
          max-width: 500px;
          margin: auto;
        }
      </style>
      <atomic-timeframe-facet>
        <element /> <!-- This element would be automatically replaced with atomic-timeframe -->
      </atomic-timeframe-facet>
    ```
  }
);
```

Reply:
That's not a bad idea honestly !

I'll file it under "to investigate" in JIRA